disnake/abc.py

```diff
@@ -1296,21 +1296,21 @@ async def send(
 
     async def send(
         self,
-        content=None,
+        content: Optional[Any] = None,
```

Comment:
`Optional[Any]` is `Any`

Reply:
Not necessarily:
```py
from typing import Any, Optional, cast

x = cast(Optional[Any], 42)
y: int = x    # error: Incompatible types in assignment (expression has type "Optional[Any]", variable has type "int")
```

The overloads also use `Optional[Any]`, which is why I used it there as well

Reply:
```python
from typing import Any, Optional

def foo(arg: Any): ...

def bar(arg: Optional[Any]): ...

foo(42)
foo(None)

bar(42)
bar(None)

# both ways work
```

Reply:
7e002c99d21d24681aaf20a341395cbd93a150b5